### PR TITLE
Update heapster to v0.20.0.alpha11

### PR DESCRIPTION
**Pull request by vishh**
This PR is needed to support disk accounting.

### CODE REVIEW

**Comment by vishh** [comment]
cc @timstclair @dchen1107

File: cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml [hunk 1]
```diff
@@ -7,26 +7,26 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v14
+  name: heapster-v15
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v14
+    version: v15
```
Comment (timstclair) [inline cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml]: Consider making `v15` and `v0.20.0-alpha11` templated parametrers (like `heapster_memory`) so we only need to change it in 1 place

**Comment by timstclair** [comment]
Thanks!

File: cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml (hunk 1 above)
Comment (vishh) [inline cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml]: That will lead to more salt ***... The version numbers will be eliminated
once we move to deployments.

**Comment by mwielgus** [comment]
LGTM

**Pull request merged**